qcelemental/util/gph_uno_bipartite.py

```diff
@@ -0,0 +1,811 @@
+"""Functions to enumerate all perfect and maximum matchings in bipartite graph.
+
+Implemented following the algorithms in the paper "Algorithms for Enumerating
+All Perfect, Maximum and Maximal Matchings in Bipartite Graphs" by Takeaki Uno,
+using numpy and networkx modules of python.
+
+NOTICE: optimization needed.
+
+Author: guangzhi XU (xugzhi1987@gmail.com; guangzhi.xu@outlook.com)
+Update time: 2017-06-29 14:41:56.
+
+Updated Dec 2017 LAB for pep8, py3, more tests, starter_match, and simpler interface
+
+"""
+from __future__ import print_function
+
+try:
+    import networkx as nx
```

Comment:
We will need to move this into the functions themselves. Also optional depends like this should be lazy loaded like `pint` is.

Reply:
right now the whole `gph_uno_bipartite.py` is conditionally loaded only if one enters the alignment code (that is, uno not in `util/__init__.py`). Prefer it always loaded and `import networkx as nx` in half-a-dozen fns over current state?

Reply:
I would always vote for importing the files so that structure is not lazy loaded. I agree its messier for programmers, but I think eliminates some lazy loading edge cases.